packages/lib/services/plugins/defaultPlugins/desktopDefaultPluginsInfo.ts

```diff
@@ -0,0 +1,13 @@
+import { InitialSettings } from '../PluginService';
+import Setting from '../../../models/Setting';
+
+export const defaultPlugins = {
+	'io.github.jackgruber.backup': '1.0.2',
+	'plugin.calebjohn.rich-markdown': '0.8.3',
+};
+
+export const initialSettings: InitialSettings = {
+	'io.github.jackgruber.backup': {
+		'path': `${Setting.value('profileDir')}`,
```

Comment:
I remember you suggested putting this into `.config` or `json` files, but we may need to use variables here, so that's why I put it in `ts` file.

Reply:
Was there a discussion about this? Sorry if I missed it or if I'm repeating something but I think it's indeed ok to have the config directly in .ts file since it's not really dynamic (it's tied to the current build).

As previously discussed however we need all data related to the default plugins we support to be in the same place.

Reply:
> Was there a discussion about this?

I had discussion with my mentor CalebJohn about this. And I will make sure we put all the default plugins related data in the same place.